File: Java/6-Pattern-Printing/76-hollow-pyramid-star-pattern.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = getInput.nextInt();

        for (int i=0; i<num; i++){
            if (i==0) {
                System.out.println(" ".repeat(num-i-1) + "*");
            }
            else if (i==num-1) {
                System.out.println("*".repeat(2*num-1));
            }
            else {
                System.out.println((" ".repeat(num-i-1) + "*") + (" ".repeat(2*i-1) + "*"));
            }
            getInput.close();
        }
    }
}

File: Python/6-Pattern-Printing/76-hollow-pyramid-star-pattern.py
num = int(input("Enter a number: "))

for i in range(num):
    if i==0:
        print(" " * (num-i-1) + "*")
    elif i==num-1:
        print("*" * (2*num-1))
    else:
        print(" " * (num-i-1) + "*" + " " * (2*i-1) + "*")